models/models.yml

```diff
@@ -0,0 +1,42 @@
+version: 2
+
+models:
+  - name: enrollment_journey_model
+    description: "This model tracks the journey of enrollments across various groups."
```

Comment:
If I understand the query correctly, it's not tracking the journey, but only getting the **current** enrolment records, right, since you are filtering by `is_current = true`?